recipes/asl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "asl" %}
```

Comment:
@pstjohn Please don't confuse ampl/mp and ASL. These are separate projects and ampl/mp includes ASL and a bunch of other things. If this recipe only provides ASL then there is probably no confusion regardless of whether it's build via `ampl/mp` or not. If it provides ampl/mp targets then I suggest naming it accordingly.